app/components/developer_primary_action_component.html.erb

```diff
@@ -8,4 +8,11 @@
     <%= inline_svg_tag "icons/solid/briefcase.svg", class: "-ml-1 mr-3 h-5 w-5" %>
     <%= t("developers.show.hire") %>
   <% end %>
+
+  <% if admin? %>
+  <button data-controller="clipboard" data-action="click->clipboard#copy click->clipboard#toggle" value="<a href='<%= developer_url(@developer) %>'><%= @developer.hero %></a>" type="button" class="shrink-0 mt-4 md:mt-0 inline-flex items-center px-4 py-2 border border-gray-300 shadow-sm text-base font-medium rounded-md text-gray-700 bg-white hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-gray-500">
```

Comment:
Is shoving this in the `value` the best approach? I wonder if it would be better to move this to a hidden element inside the controller with a target. But read my comment below on `text/html` first because that might help guide you a bit.